hudi-client/src/test/java/HoodieClientExample.java

```diff
@@ -95,7 +95,7 @@ public void run() throws Exception {
     HoodieWriteConfig cfg = HoodieWriteConfig.newBuilder().withPath(tablePath)
         .withSchema(HoodieTestDataGenerator.TRIP_EXAMPLE_SCHEMA).withParallelism(2, 2).forTable(tableName)
         .withIndexConfig(HoodieIndexConfig.newBuilder().withIndexType(IndexType.BLOOM).build())
-        .withCompactionConfig(HoodieCompactionConfig.newBuilder().archiveCommitsWith(2, 3).build()).build();
+        .withCompactionConfig(HoodieCompactionConfig.newBuilder().archiveCommitsWith(20, 30).build()).build();
```

Comment:
if we set minToKeep=2, the app will exit with `java.lang.IllegalArgumentException: Increase hoodie.keep.min.commits=2 to be greater than hoodie.cleaner.commits.retained=10. Otherwise, there is risk of incremental pull missing data from few instants`.
just set to DEFAULT_MIN_COMMITS_TO_KEEP and DEFAULT_MAX_COMMITS_TO_KEEP